src/goto-symex/goto_symex_state.cpp

```diff
@@ -275,7 +272,10 @@ ssa_exprt goto_symex_statet::rename_ssa(ssa_exprt ssa, const namespacet &ns)
     level == L0 || level == L1,
     "rename_ssa can only be used for levels L0 and L1");
   if(level == L0)
-    set_l0_indices(ssa, ns);
+  {
+    const renamedt<ssa_exprt, L0> ssa_l0 = set_l0_indices(std::move(ssa), ns);
+    ssa = ssa_l0.get();
```

Comment:
Does the temporary really add value? I think it's pretty clear what `set_l0_indices` does. Therefore I'd prefer `ssa = set_l0_indices(std::move(ssa), ns).get();`